applications/ParticleMechanicsApplication/custom_strategies/schemes/MPM_residual_based_bossak_scheme.hpp

```diff
@@ -632,7 +632,8 @@ class MPMResidualBasedBossakScheme: public Scheme<TSparseSpace,TDenseSpace>
 			
             }
             
-            //std::cout << " ============ " <<  NormVel << " ============ " << std::cout;
+            // std::cout << " ============ " <<  NormVel << " ============ " << std::cout;
```

Comment:
I would use KRATOS_INFO_IF with a conditional on the echo level

@jcotela recently created the ndof_updater, with this the code duplication of this class could be reduced

Reply:
Ahh! These are my private lines to check the parallelization. I don't need it actually. Will remove.